.rspec

```diff
@@ -1,3 +1,2 @@
 --warnings
 --color
---require spec_helper
```

Comment:
Fair enough - the point was really to be explicit about which bootstrapper is used for a given spec.